Applications/ResNet/res/resnet18.ini

```diff
@@ -0,0 +1,505 @@
+#
+# SPDX-License_Identifier: Apache-2.0
+# @file resnet18.ini
+# @author Jijoong Moon <jijoong.moon@samsung.com>
+# @date Jan 27 2021
+# @brief input configuration of Residual Net 18
+#
+
+[Model]
+Type = NeuralNetwork
+Learning_rate = 1e-4
+Epochs = 2
+Optimizer = adam
+Loss = cross
+Save_Path = "model.bin"
+batch_size = 128
+beta1 = 0.9
+beta2 = 0.999
+epsilon = 1e-7
+
+[inputlayer]
+Type = input
+Input_Shape = 3:32:32
+
+[first_layer_112_1]
+Type=conv2d
+input_layers = inputlayer
+kernel_size = 3,3
+bias_initializer = zeros
+weight_initializer = xavier_uniform
+filters=64
+stride =1,1
+padding=1,1
+
+[preparation_bn]
+Type=batch_normalization
+input_layers = first_layer_112_1
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+beta_initializer = zeros
+gamma_initializer = ones
+moving_mean_initializer = zeros
+moving_variance_initializer = ones
+
+[out0]
+Type=output
+input_layers=preparation_bn
+
+[B1_residue_1_conv_3]
```

Comment:
Can we define a block in a separate file and use it as a backbone here.
It is almost impossible to verify if this is the correct/desired architecture.

Reply:
Yes. That's what I thought too. but let's do it after finishing the evaluation.